asmcomp/cmmgen.ml

```diff
@@ -262,13 +262,45 @@ let untag_int i dbg =
   | Cop(Cor, [c; Cconst_int 1], _) -> Cop(Casr, [c; Cconst_int 1], dbg)
   | c -> Cop(Casr, [c; Cconst_int 1], dbg)
 
-let if_then_else (cond, ifso, ifnot) =
+(* Approximation of the "then" and "else" continuations in [transl_if] *)
+type then_else_approx =
+  | Cond (* then => true, else => false *)
```

Comment:
Actually, I don't find the use of the word "approximation" very helpful here.  Describing this directly in terms of how it affects the result of `transl_if` would probably be clearer.